belt-bash/src/lib.rs

```diff
@@ -0,0 +1,198 @@
+#![no_std]
+#![doc = include_str!("../README.md")]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg"
+)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![warn(missing_docs)]
+
+/// Number of 64-bit words in the state
+const STATE_WORDS: usize = 24;
+
+/// Precalculated rotation params
+const ROTATION_PARAMS: [(u32, u32, u32, u32); 8] = [
```

Comment:
You do not need to manually do this. Compiler can handle constant inlining itself perfectly well after you unrolled the rounds loop.